core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -56,8 +59,10 @@ final class ShuffleBlockFetcherIterator(
     shuffleClient: ShuffleClient,
```

Comment:
Could you update the Scaladoc to document the two new parameters here? I understand what `streamWrapper` means from context but it might be useful for new readers of this code.